src/com/xilinx/rapidwright/util/Job.java

```diff
@@ -78,14 +83,26 @@ public Pair<String,String> createLaunchScript() {
         FileTools.makeDirs(dir);
 
         startupScript.add("cd " + dir);
-        startupScript.add(getCommand() + " > " + DEFAULT_COMMAND_LOG_FILE + " 2>&1");
+        startupScript.add(getCommand() + " > " + cmdLogFile + " 2>&1");
 
-        String startupScriptName = dir + File.separator + DEFAULT_SCRIPT_NAME + scriptExt;
+        String startupScriptName = dir + File.separator + scriptBaseName + scriptExt;
         FileTools.writeLinesToTextFile(startupScript, startupScriptName);
         new File(startupScriptName).setExecutable(true);
-        String startupScriptLog = dir + File.separator + DEFAULT_SCRIPT_LOG_FILE;
+        String startupScriptLog = dir + File.separator + scriptLogFile;
         return new Pair<String,String>(startupScriptName,startupScriptLog);
     }
+
+    /**
+     * @param scriptBaseName the script's base name to set
+     */
+    public void setScriptBaseName(String scriptBaseName) {this.scriptBaseName = scriptBaseName;}
+
+
+    /**
+     * @return the script's base name
```

Comment:
Seems like this Javadoc isn't quite right

Reply:
Sorry about that. I updated it to have the right description of the return value.